framework/base/Model.php

```diff
@@ -773,7 +773,7 @@ public function activeAttributes()
         if (!isset($scenarios[$scenario])) {
             return [];
         }
-        $attributes = $scenarios[$scenario];
+        $attributes = array_unique($scenarios[$scenario]);
         foreach ($attributes as $i => $attribute) {
```

Comment:
since we are already on the subject, how about reducing this line like this? https://3v4l.org/1N2A6